training/level-2-command-line-interfaces/dragon-warrior/rwharris-nd_stdlib_cli.py

```diff
@@ -0,0 +1,19 @@
+"""

```

Comment:
The name of the file is not a valid module name via PEP8 and is invalid syntax for imports.
No dashes allowed.

:)